release_automation.sh

```diff
@@ -34,12 +34,43 @@ MOBILE_REPO="wordpress-mobile"
 
 set -e
 
+SCRIPT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPT_PATH"
+source ./release_utils.sh
+
+
+# Check if script is up-to-date
+LOCAL_COMMIT=$(git rev-parse HEAD)
+git remote update
```

Comment:
I see that the rest of Git commands executed in the script use the `execute` function, so for consistency, we could use it here too, wdyt?
```suggestion
execute "git" "remote" "update"
```

Reply:
👍  Makes sense. I missed that we were using that.